website/pages/blog/judging-open-source-by-github-stars.mdx

```diff
@@ -0,0 +1,252 @@
+---
+title: Judging Open Source by Github Stars
+tags: [open-source]
+authors: yassin
+date: 2023-05-09
+description: Exposing the dark side of open source projects
+image: /blog-assets/announcing-free-single-sign-on-for-graphql-hive/cover.png
+thumbnail: /blog-assets/announcing-free-single-sign-on-for-graphql-hive/thumbnail.png
```

Comment:
I'm not sure how you generate those thumbnails and cover pictures 🤔

Reply:
Figma! 😁 will send you a link on Slack